test/runnable/overload.d

```diff
@@ -1025,6 +1025,28 @@ void test11915()
 }
 
 /***************************************************/
+// 15674
+
+void get15674(out int i)
+{
+    i = 42;
+}
+
+struct Foo15674
+{
+    int v;
+    alias v this;
+}
+
+void test15674()
+{
+    Foo15674 f;
+    get15674(f);
+    assert(f == 42);
+    assert(f.v == 42);
+}
```

Comment:
Would it make sense to mirror [the tests for issue 7945](https://github.com/D-Programming-Language/dmd/blob/969e0903a66d06331785495cd88032fa6d19eca6/test/runnable/aliasthis.d#L1190-L1197) ?